.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,6 +4,18 @@ about: Create a report to help us improve Electron
 
 ---
 
+<!--  We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
```

Comment:
I think we should be a little more explicit here, maybe something like this?
```suggestion
<!--  As an open source project that depends on individual and corporate contributions, it can take a long time for issues to be addressed, so please be patient and we will get back to you as soon as we can.
```